### PR TITLE
14 input fields should not resemble buttons

### PR DESCRIPTION
**Pull request by svdwyer**

### CODE REVIEW

**Review by pheyvaer — CHANGES_REQUESTED** [review]
Make sure that the latest code from `main` is in the branch.

File: src/popup/popup.js [hunk 1]
```diff
@@ -156,9 +156,9 @@ const bindProfileDialogEvents = () => {
   });
   [idp, webID].forEach(i => i.addEventListener('input', inputListener));
 
-  colors.forEach(
-    (input) => input.addEventListener('change', () => avatar.style.backgroundColor = input.value)
-  );
+  // colors.forEach(
```
Comment (pheyvaer) [inline src/popup/popup.js:159]: Can't we just delete these lines then?

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: src/popup/popup.js (hunk 1 above)
Comment (svdwyer) [reply src/popup/popup.js]: correct
✔️

**Review by pheyvaer — APPROVED** [review]

**Comment by svdwyer** [comment]
fixes https://github.com/SolidLabResearch/solid-identity-manager/issues/14

**Pull request merged**